dpctl/tests/test_sycl_kernel_submit.py

```diff
@@ -278,3 +279,43 @@ def test_kernel_arg_type():
     _check_kernel_arg_type_instance(kernel_arg_type.dpctl_float64)
     _check_kernel_arg_type_instance(kernel_arg_type.dpctl_void_ptr)
     _check_kernel_arg_type_instance(kernel_arg_type.dpctl_local_accessor)
+
+
+def get_spirv_abspath(fn):
+    curr_dir = os.path.dirname(os.path.abspath(__file__))
+    spirv_file = os.path.join(curr_dir, "input_files", fn)
+    return spirv_file
+
+
+def test_submit_local_accessor_arg():
+    try:
+        q = dpctl.SyclQueue("level_zero")
+    except dpctl.SyclQueueCreationError:
+        pytest.skip("OpenCL queue could not be created")
+    fn = get_spirv_abspath("local_accessor_kernel_inttys_fp32.spv")
```

Comment:
Should we document the SYCL code that was used to generate this SPIR-V file?

Reply:
Yes, I think that's sensible

Reply:
The SPIR-V file was copied from one of the libsyclinterface tests, I've documented where it came from and added a comment